app/index.html

```diff
@@ -78,41 +78,47 @@
             </a>
           </li>
           <li>
-            <a href="./user.html">
-              <i class="material-icons">person</i>
-              <p>User Profile</p>
-            </a>
-          </li>
-          <li>
-            <a href="./table.html">
-              <i class="material-icons">content_paste</i>
-              <p>Table List</p>
+            <a href="./data.html">
+              <i class="material-icons">data</i>
+              <p>Bike Usage Data</p>
             </a>
           </li>
           <li>
-            <a href="./typography.html">
-              <i class="material-icons">library_books</i>
-              <p>Typography</p>
+            <a href="./login.html">
```

Comment:
Admin login page shouldn’t be listed on the sidebar, the user should access <domain name>/admin to go to the admin login page. Please remove this li tag.

Reply:
So how can admin login? Also if we don't want user to see the checklist either, then "usage data" button should be the only one leaves in the sidebar.